policyengine_us/variables/gov/states/nc/ncdhhs/scca/nc_scca_child_eligible.py

```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class nc_scca_child_eligible(Variable):
```

Comment:
otherwise reads as a person-level variable
```suggestion
class nc_scca_has_eligible_child(Variable):
```